packages/xstate-angular/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@xstate/angular",
+  "version": "0.1.0",
+  "description": "XState tools for Angular",
+  "keywords": [
+    "state",
+    "machine",
+    "statechart",
+    "scxml",
+    "state",
+    "graph",
+    "angular",
+    "inject"
+  ],
+  "author": "Tomasz Ducin <tomasz.ducin@gmail.com>",
```

Comment:
All other packages mention David as the author so it feels weird to have a different author here. But I don't know, an "author" field in the MIT-licensed OSS software is a weird concept to me 🤷‍♂️ Maybe we should just remove this field altogether from (almost?) all packages?